crates/primitives-traits/src/receipt.rs

```diff
@@ -20,4 +21,7 @@ pub trait Receipt:
 {
     /// Returns transaction type.
     fn tx_type(&self) -> u8;
+
+    /// Calculates the receipts root of all receipts in a block.
```

Comment:
nothing forces the receipts to belong to the same block, right?
```suggestion
    /// Calculates the receipts root of the given receipts.
```